2018-03-05 16:14:42,737 [9] INFO - 启动应用程序...
2018-03-05 16:14:42,750 [9] INFO - 启动核心服务...
2018-03-05 16:14:42,764 [9] DEBUG- 搜索插件...
2018-03-05 16:14:42,765 [9] DEBUG- 加载插件...
2018-03-05 16:14:42,825 [9] INFO - Running autostart commands...
2018-03-05 16:14:42,828 [9] DEBUG- 初始化工作台...
2018-03-05 16:14:43,069 [9] INFO - 初始化工作区...
2018-03-05 16:14:43,098 [9] INFO - Loading addin 软控信息化装备平台
2018-03-05 16:14:44,348 [9] INFO - 布局配置加载成功...
2018-03-05 16:14:44,379 [9] INFO - 创建主工具栏
2018-03-05 16:14:44,423 [9] INFO - 创建主菜单
2018-03-05 16:14:44,724 [9] INFO - 刷新插件...
2018-03-05 16:14:44,727 [9] INFO - 初始化服务、宿主管理器、界面组件...
2018-03-05 16:14:44,728 [9] INFO - 向宿主管理器中添加服务、工具箱和属性服务...
2018-03-05 16:14:44,728 [9] INFO - 初始化事件处理...
2018-03-05 16:14:44,737 [9] DEBUG- 启动工作台...
2018-03-05 16:14:44,738 [9] DEBUG- 最大连接数：50
2018-03-05 16:14:44,738 [9] DEBUG- 侦听的端口号：3666
2018-03-05 16:14:44,768 [9] DEBUG- 开始侦听...
2018-03-05 16:14:51,753 [16] ERROR- 用户 'sa' 登录失败。
2018-03-05 16:14:51,754 [16] ERROR-    在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 ICSharpCode.Data.Core.DatabaseDrivers.SQLServer.SQLServerDatabaseDriver.PopulateDatabases(IDatasource datasource)
2018-03-05 16:22:38,132 [9] DEBUG- 16
2018-03-05 16:23:50,745 [9] INFO - 启动应用程序...
2018-03-05 16:23:50,760 [9] INFO - 启动核心服务...
2018-03-05 16:23:50,776 [9] DEBUG- 搜索插件...
2018-03-05 16:23:50,777 [9] DEBUG- 加载插件...
2018-03-05 16:23:50,796 [9] INFO - Running autostart commands...
2018-03-05 16:23:50,799 [9] DEBUG- 初始化工作台...
2018-03-05 16:23:51,074 [9] INFO - 初始化工作区...
2018-03-05 16:23:51,093 [9] INFO - Loading addin 软控信息化装备平台
2018-03-05 16:23:51,847 [9] INFO - 布局配置加载成功...
2018-03-05 16:23:51,878 [9] INFO - 创建主工具栏
2018-03-05 16:23:51,910 [9] INFO - 创建主菜单
2018-03-05 16:23:52,179 [9] INFO - 刷新插件...
2018-03-05 16:23:52,182 [9] INFO - 初始化服务、宿主管理器、界面组件...
2018-03-05 16:23:52,182 [9] INFO - 向宿主管理器中添加服务、工具箱和属性服务...
2018-03-05 16:23:52,183 [9] INFO - 初始化事件处理...
2018-03-05 16:23:52,189 [9] DEBUG- 启动工作台...
2018-03-05 16:23:52,191 [9] DEBUG- 最大连接数：50
2018-03-05 16:23:52,191 [9] DEBUG- 侦听的端口号：3666
2018-03-05 16:23:52,210 [9] DEBUG- 开始侦听...
2018-03-05 16:23:57,697 [15] ERROR- 用户 'sa' 登录失败。
2018-03-05 16:23:57,699 [15] ERROR-    在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 ICSharpCode.Data.Core.DatabaseDrivers.SQLServer.SQLServerDatabaseDriver.PopulateDatabases(IDatasource datasource)
2018-03-05 16:24:00,714 [9] INFO - 加载组件属性文件...
2018-03-05 16:24:00,763 [9] INFO - 加载组件动画文件...
2018-03-05 16:24:00,774 [9] INFO - 加载组件动画文件...
2018-03-05 16:24:23,699 [9] DEBUG- 16
